src/main/java/com/powsybl/openloadflow/dc/equations/DcEquationSystemCreator.java

```diff
@@ -38,9 +38,11 @@ private void createBuses(EquationSystem<DcVariableType, DcEquationType> equation
         for (LfBus bus : network.getBuses()) {
             var p = equationSystem.createEquation(bus, DcEquationType.BUS_TARGET_P);
             bus.setP(p);
-            if (bus.isSlack()) {
+            if (bus.isReference()) {
                 equationSystem.createEquation(bus, DcEquationType.BUS_TARGET_PHI)
                         .addTerm(equationSystem.getVariable(bus.getNum(), DcVariableType.BUS_PHI).createTerm());
+            }
+            if (bus == network.getSlackBus()) {
                 p.setActive(false);
             }
```

Comment:
add a comment explaining the why only first slack bus, in line with what is done on target vector